pyproject.toml

```diff
@@ -44,6 +44,7 @@ isort = {version = "^5.7.0", extras = ["pyproject"]}
 jinja2-cli = "^0.7.0"
 mkdocs-coverage = "^0.2.1"
 mkdocs-macros-plugin = "^0.5.0"
+mkdocs-gen-files = {version = "^0.3.0", markers = "python_version>='3.7'"}
```

Comment:
Does it mean we won't be able to build docs on Python 3.6?

Reply:
Yes. I also looked into supporting 3.6 in my plugin but I confirmed that a very important feature is needed from Python 3.7, so it's not viable.

Reply:
Out of curiosity, which feature is this :stuck_out_tongue: ?

Reply:
Module-level __getattr__
https://www.python.org/dev/peps/pep-0562/
https://github.com/oprypin/mkdocs-gen-files/blob/96cc72d8b84aa0593b782424502541c736dd7e11/mkdocs_gen_files/__init__.py#L18

Reply:
I see, thanks. It's OK if the docs can't be built using Python 3.6. CI is already using 3.8 anyway.